config/00-init/03-db-deployment.yaml

```diff
@@ -30,41 +30,39 @@ spec:
     spec:
       containers:
         - name: db
-          image: postgres:13@sha256:260a98d976574b439712c35914fdcb840755233f79f3e27ea632543f78b7a21e
+          image: registry.redhat.io/rhel8/postgresql-13:1-18
```

Comment:
Do we need this change here ? We could "mutate" this, in the operator, on the OpenShift target only, couldn't we ?

Reply:
We won't need to change, this is just a PR if we want to change the image what other changes do we need. All changes will be in operator only. This image is not public so cant us upstream

Reply:
ah I see 👍🏼